server/app.js

```diff
@@ -50,4 +50,52 @@ app.use(notFound);
 // error handler
 app.use(errorHandler);
 
+// test redis
+// to start redis server in background
+// run "redis-server --daemonize yes"
+// to stop the server running "redis-cli shutdown"
+const redis = require("redis");
+const client = redis.createClient({
+  host: "127.0.0.1",
+  port: 6379,
+});
+client.on("error", (err) => {
+  console.log("Error " + err);
+});
+
+client.on("ready", function () {
+  console.log("redis is running");
+});
+
+// test bull
+var Queue = require("bull");
+
+const myFirstQueue = new Queue("my-first-queue", {
+  redis: { port: 6379, host: "127.0.0.1" },
```

Comment:
you can probably put these values into the environment file

Reply:
Yeah, you right boss.🙂